src/bin/driver.rs

```diff
@@ -226,6 +211,8 @@ mod app {
 
         let settings = Settings::default();
 
```

Comment:
Add None to the enum. Then the unwrap and the double nesting in an option go away